ui/lib/ldap/addon/components/page/library/details/accounts.hbs

```diff
@@ -4,14 +4,13 @@
       <div class="is-flex-between">
         <h3 class="is-size-5 has-text-weight-semibold">All accounts</h3>
         {{#if @library.canCheckOut}}
-          <button
-            type="button"
-            class="button is-link"
```

Comment:
For posterity - after consulting design `is-link` mapped to `@color=tertiary` + adding an icon